<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Vibrance Switcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="apps_listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="vibrance_text">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Application vibrance: </string>
    </property>
   </widget>
   <widget class="QSlider" name="def_vibrance_Slider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="def_vibrance_text">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default vibrance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="def_vibrance_level">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QSlider" name="vibrance_Slider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>400</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="active_checkBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>345</y>
      <width>55</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Active</string>
    </property>
   </widget>
   <widget class="QLabel" name="vibrance_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="add_Button">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="delete_Button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="executable_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="executable_text">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current executable:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
